index.html

```diff
@@ -0,0 +1,229 @@
+          <li>
+            Robots must fit within a 500mm ✖️ 500mm ✖️ 500mm cube at the start of each match.
+          <li>
+            Robots must fit within a 500mm ✖️ 750mm ✖️ 750mm cube, with one 750mm axis vertical,
+            at all times during each match.
+          <li>
+            Robots must not deliberately drop any part of themselves onto the arena.
+          <li>
+            Robots must have all electronics securely mounted and easily removable.
+          <li>
+            Robots must provide adequate protection to the batteries they use, including
+            protection from mechanical damage.
+          <li>
+            Robots must not have any devices designed to produce noise except where included
+            in the SourceBots kit.
+          <li>
+            Robots must not have any fiducial markers visible to other robots.
+          <li>
+            Robots must have a spare USB slot accessible to match officials.
+          <li>
+            Robots must have a power switch, obvious and on top, which will completely immobilise
+            the robot.
+          <li>
+            Robots must not have any sharp edges or points.
+        </ol>
+        <p>
+          Violations of these regulations will result in penalties, including deductions
+          of points or outright disqualification.
+        </p>
+      </div>
+    </section>
+    <section>
+      <div class="fixed-width">
+        <h2><a class="anchor" id="specifications"></a>Specifications</h2>
+        <h3><a class="anchor" id="marker-specification"></a>Markers</h3>
+        <p>
+          The arena is labelled with fiducial markers. Each marker pattern encodes a number. Each marker number is associated with a particular feature within the arena, and also has an associated size.
+        </p>
+        <table>
+          <thead>
+            <tr>
+              <th scope="col">Item</th>
+              <th scope="col">Marker Numbers</th>
+              <th scope="col">Marker Size (mm)</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Arena boundary</td>
+              <td>0 - 27</td>
+              <td>200</td>
+            </tr>
+          </tbody>
+        </table>
+        <p>
+          Two sets of marker codes will be used: one for development purposes, and one for the competition itself. The competition set is only to be used inside the SourceBots arena at the SourceBots competition. This is so that people carrying markers past the arena do not confuse robots. The competition codes are 100 above the development codes. When run in competition mode, the software provided by SourceBots will subtract 100 from the detected marker codes, as well as ignore the development codes. Markers outside of the ranges used in the game will also be ignored.
```

Comment:
Question: Do we use competition markers at smallpeice? Previous rules don't mention it, so I presume not?

Reply:
No, sbot doesn't seen to have any marker offset support.